src/navigator.js

```diff
@@ -0,0 +1,236 @@
+import React, { Component } from 'react';
+import { observer } from 'mobx-react';
+import createBrowserHistory from 'history/createBrowserHistory';
+import { Router, Route } from 'react-router-dom';
+import { syncHistoryWithStore } from 'mobx-react-router';
+import Container from './component/container';
+import { NotificationBar } from './component/notification';
+import Welcome from './view/welcome';
+import Loader from './view/loader';
+import Seed from './view/seed';
+import SeedVerify from './view/seed-verify';
+import SeedSuccess from './view/seed-success';
+import SetPassword from './view/set-password';
+import Password from './view/password';
+import NewAddress from './view/new-address';
+import Home from './view/home';
+import Payment from './view/payment';
+import PayLightningConfirm from './view/pay-lightning-confirm';
+import PayLightningDone from './view/pay-lightning-done';
+import PayBitcoin from './view/pay-bitcoin';
+import PayBitcoinConfirm from './view/pay-bitcoin-confirm';
+import PayBitcoinDone from './view/pay-bitcoin-done';
+import Invoice from './view/invoice';
+import InvoiceQR from './view/invoice-qr';
+import Deposit from './view/deposit';
+import Channel from './view/channel';
+import ChannelDetail from './view/channel-detail';
+import ChannelDelete from './view/channel-delete';
+import ChannelCreate from './view/channel-create';
+import Transaction from './view/transaction';
+import Setting from './view/setting';
+import SettingUnit from './view/setting-unit';
+import SettingFiat from './view/setting-fiat';
+import Notification from './view/notification';
+import TransactionDetail from './view/transaction-detail';
+import {
+  nav,
+  wallet,
+  payment,
+  invoice,
+  channel,
+  transaction,
+  setting,
+} from './action';
+import store from './store';
+
+const history = syncHistoryWithStore(createBrowserHistory(), store.route);
+
+class Navigator extends Component {
+  render() {
+    const { lastNotification, displayNotification } = store;
+    return (
+      <Router history={history}>
```

Comment:
Nice how this enables the history feature & looks much nicer than the previous `main.js`